lib/vfscore/stdio.c

```diff
@@ -211,15 +212,28 @@ static struct vfscore_file  stdio_file = {
 	.f_ep = UK_LIST_HEAD_INIT(stdio_file.f_ep)
 };
 
-void init_stdio(void)
+int init_stdio(void)
 {
-	int fd __maybe_unused;
+	int fd;
 
 	fd = vfscore_alloc_fd();
-	UK_ASSERT(fd == 0);
+	if (fd != 0) {
+		uk_pr_crit("failed to allocate fd for stdout\n");
```

Comment:
```suggestion
		uk_pr_crit("failed to allocate fd for stdin\n");
```

Isn't fd == 0 stdin, or am I missing something?

Reply:
You are right, this was wrong already in the previous code and I did somehow not notice. I am updating this. :+1: